src/wf/device.hpp

```diff
@@ -34,6 +35,13 @@ void computePredicate(float *data,
 #endif
 }
 
+PERFFASPECT()
+void computePredicateDeltaUQ()
+{
+  std::runtime_error(
+      "Computing DeltaUQ predications on device is not supported yet");
```

Comment:
Why not use the introduced `THROW` Macro?

Reply:
Done